tests/test_brand_jwt_auth__verify_brand.py

```diff
@@ -34,30 +29,6 @@ def test__verify_brand__where_user_has_brand(
 
         # Assert
         mock_set_brand_to_context.assert_called_once_with("mybrandone")
-        mock_set_brand_alias_to_context.assert_called_once_with(None)
-
-    @mock.patch("sag_py_auth_brand.brand_jwt_auth.set_request_brand_alias_to_context")
-    @mock.patch("sag_py_auth_brand.brand_jwt_auth.set_request_brand_to_context")
-    def test__verify_brand__where_brand_is_missing(
-        self, mock_set_brand_to_context: Mock, mock_set_brand_alias_to_context: Mock
-    ) -> None:
-        # Arrange
-        brand_jwt_auth: BrandJwtAuth = build_sample_jwt_auth(["myEndpoint"])
-
-        resource_access: Optional[Dict[str, Any]] = {"role-brand": {"roles": ["mybrandone", "mybrandtwo"]}}
-
-        token: Token = get_token(None, resource_access)
-
-        # Act
-        with pytest.raises(HTTPException) as exception, self.assertLogs(level="DEBUG") as log_watcher:
-            brand_jwt_auth._verify_brand(token, "mybrandthree")
-
-        # Assert
-        self.assertTrue(LogMessages.MISSING_BRAND_ALIAS in "".join(log_watcher.output))
-        assert exception.value.status_code == 403
-        assert exception.value.detail == "Missing brand."
```

Comment:
should not be removed entirely due to the missing brand check

Reply:
fixed with https://github.com/SamhammerAG/sag_py_auth_brand/pull/7/commits/42b37f62d34ec7f7b06334a215e9923ca8f282a7